upb/msg.int.h

```diff
@@ -0,0 +1,603 @@
+/*
+** Our memory representation for parsing tables and messages themselves.
+** Functions in this file are used by generated code and possibly reflection.
+**
+** The definitions in this file are internal to upb.
+**/
```

Comment:
every file should start with a copyright notice.  Additionally, you should create this file as a git cp of the old one so that the diff tracking and blame are better

Reply:
I'll follow up with a change to add copyright notices to all files.

Git doesn't store rename information, it's automatically detected after the fact. See: https://stackoverflow.com/a/7941544

> Git tracks file contents, not filenames. So renaming a file without changing its content is easy for git to detect. (Git does not track, but performs detection; using git mv or git rm and git add is effectively the same.)